### DISCUSSION TITLE
Meeting showing as inactive when it shouldn't do

### DISCUSSION THREAD

**Discussion by dualdiagnosishub** [code4recovery/12-step-meeting-list#490] [Q&A] [answered]
**Describe the bug**
A meeting in my directory is displaying "Temporarily Inactive" but it shouldn't do.
Is also having issues with saving the meetings after I've edited them or made any changes.

Web Host has upgraded the PHP version used - when is this issue going to be fixed to support newer versions of PHP because the plugin is unusable at the moment?

**To Reproduce**
1. Go to: https://www.ddauk.org/meetings/hows-your-head-topic-discussion-meeting-face-to-face/
2. Meeting is showing "Temporarily Inactive" when it shouldn't do

**Expected behavior**
Not display: "Temporarily Inactive".
Allow me to save whenever I've made changes to the listing.

**Screenshots**
https://paste.pics/DF942

**Desktop (please complete the following information):**
MacOS Big Sur: 11.5.1 (20G80)
Safari: Version 14.1.2 (16611.3.10.1.3)

**Smartphone (please complete the following information):**

**Additional context**

**Comment by tech2serve** [comment]
What version of PHP is installed on your site? If it is the 7.x series, I’m confident TSML works just fine with it. If you have PHP errors or warnings, please post them here in a response to this thread. 

Regarding your concern about the meeting showing as inactive, please provide a screen cap of your edit meeting page (which you can add directly in your response using the GitHub web interface, not email). In particular, I want to see how you’ve answer the question about attending the meeting in-person (should be set to “yes”) and the Location and Address fields.

**Comment by joshreisner** [comment, marked as answer]
Hi @nextlevelwebdesign to give a little context on what is going on: the plugin uses addresses to distinguish between online and in-person meetings, and it allows you to use approximate addresses, like London, or England, for online meetings.

This is relevant because it looks like the address for your meeting is "Africa" which the plugin, understandably, considers an approximate location. Having a specific address or geographic point is necessary for the plugin to consider it an in-person meeting.

There is a secondary problem which is that we use Google to get the corrected address for every meeting, this store inside a shopping mall is a tricky one.

I recommend using `Ariel Way, London W12 7GF, UK` as the address for the meeting, I think this will produce the result you want.

PHP version should not be a factor here.

**Reply by joshreisner** [reply]
Another address that seems to work for me is `West 12 8PP, Westfield London Shopping Centre, Ariel Way, London W12 8PN, UK`

**Comment by dualdiagnosishub** [comment]
Hi

That worked and solved the problem. It was my mistake again and not a bug!

Thanks for all your help…

Kind Regards,
Jon


From: Josh Reisner ***@***.***>
Date: Wednesday, 4 August 2021 at 16:56
To: code4recovery/12-step-meeting-list ***@***.***>
Cc: Next Level Web Design CIC ***@***.***>, Mention ***@***.***>
Subject: Re: [code4recovery/12-step-meeting-list] Meeting showing as inactive when it shouldn't do (#490)

Another address that seems to work for me is West 12 8PP, Westfield London Shopping Centre, Ariel Way, London W12 8PN, UK

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/code4recovery/12-step-meeting-list/discussions/490#discussioncomment-1130928>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASNCLWJJ4P27D67I3Z2CR5LT3FPLVANCNFSM5BRGE7OA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.

**Comment by jmh2911** [comment]
This update isn't working me. I still have groups showing as temporarily inactive. Some groups don't use the supported providers. And some groups don't include the conference link for privacy purposes. Is there something else I need to do? Can I just reupload the previous version? Thank you!
https://liferecoverygroups.com/meetings/lose-it-for-life-online-3/

**Reply by joshreisner** [reply]
@jmh2911 you're welcome of course to roll back to any version of the plugin you wish, there are [instructions online](https://www.google.com/search?client=safari&rls=en&q=roll+back+wordpress+plugin+update) for how to do that

if there are conferences services that should be added please start a new discussion topic and tell us about them (what they're called, what the URL structure looks like)

it's also possible to add your own custom conference services to expand the definition of what's "online" -- if you want to do that it looks like it's not documented so please start a new discussion thread and we can help you

we made the decision a long time ago that if a meeting is not joinable it should not be considered "online." (this is consistent with our position that meetings without a street address should not be considered "in-person." this plugin is for listing public recovery meetings, so meetings that don't want to be listed publicly could still be listed in a more vague way on another page on your website.) this version only makes that distinction more obvious with red text, but previous versions used graying out to convey the same info.

**Comment by jmh2911** [comment]
Thank you! I figured out my problem. As a workaround, I just needed to add a phone number in the online notes section. Thank you!
